samples/isolated-unit-tests/.gitignore

```diff
@@ -0,0 +1,264 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Do I have to include this file? I included it becasueI see some other sample has this,  but I feel like it's a little redundant as this supposed to be a auto-generated file.